src/server/validation/paymentCode.js

```diff
@@ -1,6 +1,6 @@
 import { check } from 'express-validator/check';
 
 export default [
-  check('payment_code').isLength({ min: 16, max: 16 }),
-  check('payment_code').trim().isHexadecimal(),
+  check('payment_code').isLength({ min: 5, max: 18 }),
+  // check('payment_code').trim().isHexadecimal(),
```

Comment:
Could we put some conditional logic around this? Maybe add custom validator for base 36 based on a simple regex?